dependencies/pom.xml

```diff
@@ -0,0 +1,80 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.google.cloud.tools.opensource</groupId>
+  <artifactId>dependencies</artifactId>
```

Comment:
maybe call this `dependency-printer`?

Reply:
Printing and the main method are mostly for experimentation. I'm not sure they'll survive in the final product. But the name could certainly be improved. Any other suggestions?

Reply:
Remember to update this too

Reply:
Forager, Navigator, Forester

Reply:
- Forager: this seems a little opaque...
- Navigator: this implies a UI, which doesn't seem to match
- Forester: this seems even more opaque...

Glad to see ideas though!

Reply:
A forester managers lots of (dependency) trees :-)